gevent/_socket2.py

```diff
@@ -396,5 +396,16 @@ def fromfd(*args):
 else:
     __implements__.remove('fromfd')
 
+if hasattr(__socket__, 'ssl'):
```

Comment:
A number of third-party libraries depend on 'socket.ssl'; I believe I encountered the issue when doing functional testing. Lying around on my machine I have versions of 'boto' (the Amazon library), 'httplib2', and 'superlance' (part of supervisord) that use it for starters.